dynamicweb/settings/base.py

```diff
@@ -190,6 +190,8 @@ def bool_env(val):
     # ungleich
     ('blog_ungleich.html', gettext('Blog')),
     ('page.html', gettext('Page')),
+    # dcl
+    ('datacenterlight/faq.html', gettext('DCL.Faq')),
```

Comment:
I suggest to name the template "Data Center Light" and not specific to the FAQ -- so that we have a template with title + text that we can use for content that is DCL related.